docs/user/getting_started.rst

```diff
@@ -23,7 +23,19 @@ easily be installed through anaconda.
 Installation
```

Comment:
I think you can suggest miniconda for those that want to use command line to install and use less space and anaconda for those who are not sure.

Reply:
Good idea. I have added the following sentence:
> Miniconda is a lightweight alternative to Anaconda if storage is a concern.